### PR TITLE
Gf/max gas fee section

### PR DESCRIPTION
**Pull request by George-field**
# [Max Gas Fee Component][(https://shortcutLink](https://app.shortcut.com/oazo-apps/story/4652/max-gas-fee-section))

  
## Changes 👷‍♀️
Added a new max gas fee component
modified the existing Item component to account for different props
  
## How to test 🧪
  Add the following code to a view with a sidebar

```
 <MaxGasPriceSection />
```

### CODE REVIEW

File: components/infoSection/Item.tsx [hunk 1]
```diff
@@ -56,7 +72,7 @@ export function Item({ label, dropdownValues, value, secondaryValue, isLoading }
               <AppSpinner />
             ) : (
               <>
-                {`${value}`}
+                {value && <>{typeof value === 'object' ? value : `${value}`}</>}
```
Comment (johnnieskywalker) [inline components/infoSection/Item.tsx:75]: [minor] How about `value instanceof Object` ? 
`typeof value === 'object'` would return `true` also if value is `null`. 
However I'm not sure which is better as result of both would be very similar.

File: components/infoSection/Item.tsx [hunk 2]
```diff
@@ -89,9 +105,25 @@ export function Item({ label, dropdownValues, value, secondaryValue, isLoading }
         </Box>
       </Flex>
       {open && (
-        <Grid as="ul" gap={2} sx={{ p: 0, m: 0, pl: 3, mt: 2, listStyle: 'none' }}>
-          {dropdownValues && dropdownValues.map((item) => <Item {...item} />)}
-        </Grid>
+        <>
+          {subLabel && (
+            <Text
```
Comment (piekczyk) [inline components/infoSection/Item.tsx:110]: add semantics `as="..."`

**Review by piekczyk — CHANGES_REQUESTED** [review]

File: components/infoSection/Item.tsx (hunk 1 above)
Comment (George-field) [reply components/infoSection/Item.tsx:75]: Ah, I was trying to find if there was something like ```React.isValidElement``` I will use that, thanks :)

**Review by piekczyk — APPROVED** [review]

File: components/infoSection/Item.tsx (hunk 1 above)
Comment (johnnieskywalker) [reply components/infoSection/Item.tsx]: gj

**Pull request merged**